src/seqdisp.cpp

```diff
@@ -871,35 +869,33 @@ static bool seq_AddTextFromFile(const char *pTextName, SEQ_TEXT_POSITIONING text
 	}
 
 	pTextBuffer = fileLoadBuffer;
-	pCurrentLine = strtok(pTextBuffer, seps);
-	while (pCurrentLine != nullptr)
+	std::istringstream stream(pTextBuffer);
+	std::string pCurrentLine;
+
+	while (std::getline(stream, pCurrentLine))
```

Comment:
It may be better to explicitly specify the delimiter here:
```suggestion
	while (std::getline(stream, pCurrentLine, '\n'))
```